samples/bluetooth/peripheral_hr_coded/src/main.c

```diff
@@ -0,0 +1,199 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/printk.h>
+#include <sys/byteorder.h>
+#include <zephyr.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+#include <bluetooth/services/bas.h>
+#include <bluetooth/services/hrs.h>
+
+static struct k_work start_advertising_worker;
+
+static struct bt_le_ext_adv *adv;
+static struct bt_conn *default_conn;
+
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	BT_DATA_BYTES(BT_DATA_UUID16_ALL, 0x0d, 0x18, 0x0f, 0x18, 0x0a, 0x18),
+};
+
+static void connected(struct bt_conn *conn, u8_t conn_err)
+{
+	if (conn_err) {
+		printk("Connection failed (err %d)\n", conn_err);
+		return;
+	}
+	
+	default_conn = bt_conn_ref(conn);
+	printk("Connected\n");
```

Comment:
if you enable CONFIG_BT_USER_PHY_UPDATE, you can use bt_conn_get_info to get the current PHY of the connection, it should return LE Coded PHY at this point.